frontend/src/scenes/sessions/Sessions.scss

```diff
@@ -13,6 +13,7 @@
 .session-player {
     padding: $default_spacing * 2 $default_spacing 0;
     height: calc(100vh - #{$top_nav_height});
```

Comment:
So the scroll bars are caused by this.

.ant-drawer-body has padding 24px, back button has height 19px (currently) and this has 32px padding.

I think the correct height here is `calc(100vh - 64px - 48px)`

Reply:
Addressed in #2659